early-init.el

```diff
@@ -26,10 +26,11 @@
 (push '(tool-bar-lines . 0) default-frame-alist)
 (push '(menu-bar-lines . 0) default-frame-alist)
 (push '(vertical-scroll-bars) default-frame-alist)
-(push '(background-color . "#232635") default-frame-alist)
-(push '(foreground-color . "#FFFFFF") default-frame-alist)
 (push '(mouse-color . "white") default-frame-alist)
 
+;; Loads a nice blue theme, avoids the white screen flash on startup.
+(load-theme 'deeper-blue t)
```

Comment:
I wonder how well this will work with Emacs running as a daemon?

Reply:
Maybe place a guard (test) there and only load when `display-graphic-p` and/or not `daemonp`?

Reply:
Works fine. I usually run emacs as a deamon and then use emacsclient to start emacs.

Reply:
Test it with `emacs -nw` and `emacs-client -c`  in a terminal emulator should to verify that it works.

Could probably use a customize variable to make it easy to change?